.gitignore

```diff
@@ -8,3 +8,4 @@
 .node-version
 npm-debug.log
 .idea
+.vscode
```

Comment:
This change seems to be unrelated to the fix

Reply:
Sorry for my mistake. I just removed it in 07a14a8